mscore/dragdrop.cpp

```diff
@@ -540,6 +541,9 @@ void ScoreView::dropEvent(QDropEvent* event)
                         }
                         event->acceptProposedAction();
                         break;
+                  case ElementType::SLUR:
```

Comment:
What about some other element types? Don't they need the same clearing?

Reply:
As far as I know, SLUR is the only ElementType that will put the ScoreView into ViewState::EDIT upon drag/drop. But your point is well taken, and I have found a better way to do this.